react/react1/week1/todo/src/App.js

```diff
@@ -0,0 +1,31 @@
+import logo from "./logo.svg";
+import "./App.css";
+import Description from "./components/Description";
+
+function App() {
+  const lists = [
+    { description: "Get out of bed", deadlineDate: "Wed Sep 13 2017" },
+    { description: "Brush teeth", deadlineDate: "Thu Sep 14 2017" },
+    { description: "Eat breakfast", deadlineDate: "Fri Sep 15 2017" },
+  ];
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <h1> ToDo-app- with-react </h1>
+      </header>
+      <div>
+        {lists.map((list) => {
+          return (
+            <Description
+              description={list.description}
```

Comment:
If you haven't already, check Object destructuring. So you might access `description` and `deadlineDate` directly.